### ISSUE TITLE
PATH doesn't include gzip or pg_dump commands during cron

### ISSUE THREAD

**Issue by patcon** [damm/backup#34]
Hey @damm, looking for advice on whether to handle this in my setup or try to get it solved in the main cookbook.

### Expectatio
I would expect that I could backup postgresql database and use gzip compression and it would work running via cron without special debugging.

### Actual observations
The cron runs silently fail, as utilities backup expects are unavailable on the PATH, which in our case in only the embedded bin dir.

### Possible solutions
1. Add some standard dirs to the cron PATH
2. Allow setting absolute paths of utilities in config.rb

The PATH approach would seem to be resolves by adding `/bin` and `/usr/bin`, at least in my case.

The global `config.rb`` approach is solved as below for me, but probably varies lots more for others:

```rb
Utilities.configure do
  gzip    '/bin/gzip'
  pg_dump '/usr/bin/pg_dump'
  cat     '/bin/cat'
  tar     '/bin/tar'
end
```

I prefer the PATH approach to ensure others can use the cookbook easily. Thoughts?

**Comment by damm** [comment]
Hmm, it's likely tripping up on this.

```
    if new_resource.cron_path
      path new_resource.cron_path
    else
      path node['languages']['ruby']['bin_dir']
    end
```

I've not seen it 'break' before myself; on Ubuntu with gzip in `/bin/gzip`.  But I can see the path logic I am using here could be bad.

If you add cron_path "/usr/bin:/bin" does it work as you expect?

**Comment by patcon** [comment]
I ended up going the following, thinking it was the minimum required:

```rb
cron_path "/usr/bin:/bin:/opt/chef/embedded/bin"
```

I haven't tried your suggestion, but wouldn't that render the `backup` command missing?

**Comment by damm** [comment]
@patcon it would be helpful; if you have a system version of ruby installed you would need whatever path it installs the binstubs into in your path.

**Comment by patcon** [comment]
Ah, I thought `backup` was being installed there due to its installation via `chef_gem`, but just realize it's installed via `gem_package` (which presumably puts it there when no system ruby).

Would `/usr/bin:/bin:/usr/local/bin:/opt/chef/embedded/bin` be a pretty good catch-all? (assuming `/usr/local/bin` is often where system gem binstubs are installed?)

**Comment by damm** [comment]
I was thinking more under the lines of 

```ruby
cron_path "/usr/bin:/bin/usr/local/bin:/opt/chef/embedded/bin:#{node['languages']['ruby']['bin_dir']}"
```

**Comment by damm** [comment]
Closing this as it should be fixed by #35
